modules/gapi/include/opencv2/gapi/gproto.hpp

```diff
@@ -12,7 +12,7 @@
 #include <vector>
 #include <ostream>
 
-#include <opencv2/gapi/util/variant.hpp>
+#include <opencv2/gapi/util/optional.hpp>
```

Comment:
I'd keep the `variant` include here. We shouldn't rely on the fact that `optional` is implemented atop of that and also includes that header - it may change one day, occasionally breaking this code.

Reply:
yes, should revert it back - added by mistake